<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-RGFDNS0" failures="1" tests="1" name="NegativeTests" time="12.243" errors="0" timestamp="2023-09-10T12:15:23 IST">
  <testcase classname="com.herokuapp.theinternet.NegativeTests" name="incorrectUsernameTest" time="12.182">
    <failure type="java.lang.AssertionError" message="Actual Error message does not contain expected. 
Actual :Your username is invalid!
×
Expected:Your password is invalid! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Actual Error message does not contain expected. 
Actual :Your username is invalid!
×
Expected:Your password is invalid! expected [true] but found [false]
at com.herokuapp.theinternet.NegativeTests.incorrectUsernameTest(NegativeTests.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- incorrectUsernameTest -->
  <testcase classname="com.herokuapp.theinternet.NegativeTests" name="incorrectPasswordTest" time="0.0">
    <ignored/>
  </testcase> <!-- incorrectPasswordTest -->
</testsuite> <!-- NegativeTests -->
